src/api/types/LoginResponse.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Italia PagoPA Proxy
+ * Cittadinanza Digitale PagoPA services
+ */
+
+import { IRestfulObject } from "../../types/BaseResponseApp";
+
+export interface ILoginResponse extends IRestfulObject {
```

Comment:
we should be able to auto-generate io-ts based code for all these models from the wallet specs, like we do [here](https://github.com/teamdigitale/italia-app/pull/143/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R15) (see [gen-api-models](https://github.com/teamdigitale/italia-utils#gen-api-models))